lib/howl/interactions/file_selection.moon

```diff
@@ -67,7 +60,18 @@ class FileSelector
 
   _chdir: (directory, text) =>
     @directory = directory
-    matcher = file_matcher self.directory_reader(directory), directory, @opts.allow_new
+    local matcher
+    if @show_subtree
+      items, timed_out = self.subtree_reader(directory, timeout: 1)
```

Comment:
Updated to 3s for now. Eventually it would be nice to show something after 1s (e.g. 'please wait..') so the user is not left wondering, but that would have to be implemented across all interactions.

We can make this a config option if needed later.